### ISSUE TITLE
[Bug] Viewing large response in raw mode stunned Insomnia

### ISSUE THREAD

**Issue by weeix** [getinsomnia/insomnia#2035]
**Describe the bug**
Sending a request to an endpoint that returns a large response (in my case it's a 1.8MB doc file) caused Insomnia to become unresponsive for quite a while. Users can only access the main menu (Application, Edit, View, etc.) and the pop-up. Closing the app while it's unresponsive doesn't help since the response would be loaded again at startup.

**To Reproduce**
1. Add a new GET request to an endpoint that would return a large response e.g. https://s7dcb5a974440757e.jimcontent.com/download/version/1389722772/module/5436503859/name/picture%20dictation%20gender1.doc
2. Press "Send".
3. Change the preview mode from "Visual Preview" to "Raw Data".
4. Insomnia would become unresponsive. A few pop-up saying "Insomnia has become unresponsive. Do you want to reload?" might appear.
5. Try to close the app while it's unresponsive.
6. Re-open the app and see that Insomnia is still unresponsive.
7. Wait for the response to load then delete it, or press "Help > Show App Data Folder" and delete the large response (or every response) inside "Insomnia/responses".

**Expected behavior**
If the response is too large, inform users instead of displaying it immediately. For example, "The response body is too big to be displayed" or "The response is too big. This might cause Insomnia to be unresponsive. [Display it anyway]" 

**Screenshots**

![image](https://user-images.githubusercontent.com/6592791/79116817-a3444600-7db3-11ea-9f3c-756af55380ab.png)

**Desktop (please complete the following information):**
 - OS: Windows 10 Home version 1903 build 18362.657
 - Installation Method: exe file
 - App Version v7.1.1

**Additional context**
This issue is a duplicate of #1127 which were closed (while being unresolved) by the stale bot.

**Comment by Lonli-Lokli** [comment]
Similar issue happens when Save to file button pressed - seems like this file save block UI as well.

**Comment by panchalkalpesh** [comment]
Verified on:
```
macOS Mojave 10.14.3
Insomnia version 7.1.1 (7.1.1.3350)
```

![Screenshot 2020-05-16 at 4 07 59 PM](https://user-images.githubusercontent.com/1808686/82129338-dd787d00-978f-11ea-8fb1-ac4d7a6528cf.png)

**Comment by heiybb** [comment]
**Describe** 
Similar issue here while opening PDF file in raw, when I clicked the Raw option at the Response section, the app got frozen, then I have to restart it, but it won't work, everytime launching the app, it will get frozen at the previous page I closed before. The CPU usage will also raise up the same time(20% usage for 8750H).
Around 5 mins later, the app recovered from the frozen status and automatically jumped to another request, but once I clicked the request (that crash the app before), it will get stuck again.

**How I get rid of the infinite boot stuck loop:**
- Wait it recovery and delete the problem request(that crash the app)
**Or**
- Go the insomnia's data dir and delete the problem request manually from `insomnia.Request.db`
(For windows user it locates at `C:\Users\yourusername\AppData\Roaming\Insomnia`)

**Env**
- OS: Windows 10 Pro Version 1909 build 18363.836
- Installation Method: exe
- App Version v7.1.1

**Comment by wc-matteo** [comment]
same here with this url: https://www.iutbayonne.univ-pau.fr/~klevron/mm/api/download?path=images/Desert.jpg

**Comment by incompletude** [comment]
I have been experiencing the same problem on Mac OS Catalina lastest with big payloads. It becomes unusable.

**Comment by rafamerlin** [comment]
Thanks @heiybb , I tried to import a huge curl request and the app got unresponsive but since that was the last request I used it would always get unresponsive and I couldn't open it at all. Deleting it from the request.db solved the issue for me.

My issue happened because instead of pasting the huge curl script I had in the URL I ended up pasting in one of the Query named properties and it froze the app forever. Pasting it on the proper URL works fine :)
